ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/button/ButtonComponentView.kt

```diff
@@ -30,15 +35,27 @@ import com.revenuecat.purchases.ui.revenuecatui.components.stack.StackComponentV
 import com.revenuecat.purchases.ui.revenuecatui.components.style.ButtonComponentStyle
 import com.revenuecat.purchases.ui.revenuecatui.components.style.StackComponentStyle
 import com.revenuecat.purchases.ui.revenuecatui.components.style.TextComponentStyle
+import com.revenuecat.purchases.ui.revenuecatui.extensions.conditional
+import kotlinx.coroutines.launch
 
 @Composable
 internal fun ButtonComponentView(
     style: ButtonComponentStyle,
     modifier: Modifier = Modifier,
 ) {
+    val coroutineScope = rememberCoroutineScope()
+    var isClickable by remember { mutableStateOf(true) }
     StackComponentView(
         style.stackComponentStyle,
-        modifier.clickable { style.actionHandler(style.action) },
+        modifier.conditional(isClickable) {
```

Comment:
This doesn't actually have any visual feedback of it being disabled. We might want to pass this state downwards however, like if we want to have a spinner instead of the actual "StackComponentView" for example... But I thought this was better than nothing for now.